campusmap/src/info/Info/Info.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { ReactElement } from 'react';
+import { Card, Col, Row, Carousel, Jumbotron, Button, Container, Form } from 'react-bootstrap';
```

Comment:
Based on the style guide, these should each be on their own line and alphabetized